docs/tarfs.md

```diff
@@ -24,21 +24,21 @@ $ nerdctl run --snapshotter nydus --rm nginx
 
 # Show mounted rootfs a container
 $ mount
-/dev/loop17 on /var/lib/containerd-nydus/snapshots/7/mnt type erofs (ro,relatime,user_xattr,acl,cache_strategy=readaround)
```

Comment:
This, and any other similar change is just to complete the transition started in the earlier PR.